Meta/Lagom/Tools/CodeGenerators/LibWeb/BindingsGenerator/IDLGenerators.cpp

```diff
@@ -3001,14 +3001,22 @@ static void collect_attribute_values_of_an_inheritance_stack(SourceGenerator& fu
                 attribute_generator.set("attribute.cpp_name", attribute.name.to_snakecase());
             }
 
-            if (attribute.extended_attributes.contains("Reflect")) {
-                auto attribute_name = attribute.extended_attributes.get("Reflect").value();
-                if (attribute_name.is_empty())
-                    attribute_name = attribute.name;
-
-                attribute_generator.set("attribute.reflect_name", attribute_name);
-            } else {
-                attribute_generator.set("attribute.reflect_name", attribute.name.to_snakecase());
+            auto reflected_attributes = {
+                "Reflect",
+                "ReflectURL",
+                "ReflectPositive",
+                "ReflectNonNegative",
```

Comment:
This commit adds new reflection attributes but the commit message says nothing about that. I'd expect the introduction of new reflection attributes to be accompanied by their implementation.